installone.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
Why is this file here? It's going to be wrong in about 3 seconds. I don't think it should be checked in story_00 but honestly I'm mostly complaining in order to test out this code review tool so feel free to ignore this. :)